pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/NotificationJobTest.java

```diff
@@ -133,12 +133,8 @@ public void getUnsentKits() {
     }
 
     @Test
-    @Ignore("Throws that error when run at ALL Tests, not if run alone! Caused by: java.security.cert.CertificateException: "
```

Comment:
I think this test really doesn't test anything.  I've uncommented it to see what happens.  It might better be removed at this point, but let's see what happens in circleCI.